src/ft2font.cpp

```diff
@@ -601,9 +601,7 @@ void FT2Font::set_text(
     FT_Bool use_kerning = FT_HAS_KERNING(face);
     FT_UInt previous = 0;
 
-    glyphs.resize(0);
-    pen.x = 0;
-    pen.y = 0;
+    clear();
```

Comment:
Note for the future reviewer: this additionally calls FT_Done_Glyph on each element of glyph.